hyperjumptech/monika
# [663/docs]: Send data to Algolia and Show search bar and search resul…

…t in the website

# Monika Pull Request (PR)  

## What feature/issue does this PR add  
1. Send data to Algolia before deploying the docs
2. Show search bar and search result in the website

## How did you implement / how did you fix it  
1.  install algoliasearch, @algolia/autocomplete-js, @algolia/autocomplete-theme-classic, react-instantsearch-dom
2. create user for api key in website algolia
3. change config apiKey in siteConfig
4. create search bar in file /SearchInput.js

## How to test  
1.  npm install algoliasearch react-instantsearch-dom
2. npm install @algolia/autocomplete-js 
3. npm install @algolia/autocomplete-theme-classic
5. search "old or rais" in search bar, then the search results appear
